src/clr-angular/forms/radio/radio-container.ts

```diff
@@ -19,13 +18,13 @@ import { NgControlService } from '../common/providers/ng-control.service';
   template: `
     <ng-content select="label"></ng-content>
     <label *ngIf="!label && addGrid()"></label>
-    <div class="clr-control-container" [ngClass]="controlClass()">
-      <div [class.clr-radio-inline]="clrInline">
-        <ng-content select="clr-radio-wrapper"></ng-content>
+    <div class="clr-control-container" [class.clr-control-inline]="clrInline" [ngClass]="controlClass()">
```

Comment:
Don't `[class.xxx]` and `[ngClass]` conflict on the same element? I remember having this discussion, I think the conclusion was that even if `ngClass` doesn't overwrite existing classes today, it's pretty unstable and we shouldn't rely on it.

Reply:
They do not conflict in this case, though I didn't want to modify the controlClass service. I don't remember having any discussions about this, but the alternative is to move the logic up to the `controlClass()` logic.

Reply:
Yes except I'll need this logic in checkboxes too so I've already consolidated it in service.